### ISSUE TITLE
Tooltips don't appear until mouse position changes

### ISSUE THREAD

**Issue by hgesler** [IronWillGames/Dransik#390]
CTRL hover tooltips on items do not appear until the mouse changes position/moves over the item.

1. Place mouse over item whose tooltip you want to view and do not move it
2. CTRL to bring up tooltip.  The pointer changes to a ? pointer. Notice nothing happens.
3. Move the pointer off/onto the item, keeping CTRL pressed
4. Notice the tooltip is drawn.

I would expect that it should draw when CTRL is pressed while over an item. Minor issue but feels like a bug.

**Comment by davidjreese** [comment]
Yeah, this is a know issue. I am not sure this is easily fixable. The mouse has to create an event to make the tooltip appear when its in the window.
